### DISCUSSION TITLE
Field numbers

### DISCUSSION THREAD

**Discussion by TinMen6** [Courseplay/Courseplay_FS22#1045] [General]
"To get the field numbers back, go into ..\My Games\FarmingSimulator2022 and open the gameSettings.xml look for 0 and set the number back to 0."

Sorry i just don't get what i should do to get the numbers back, could someone help me out a bit, english is not my native language so that might be the problem.

**Comment by Knuspexs** [comment]
You need to change in the gameSettings.xml the value from 2 to 0 for the entry `<ingameMapFilters>0</ingameMapFilters>`

**Reply by TinMen6** [reply]
Oh ok. I was unclear which line should i change. Big thanks for the help

**Reply by Tensuko** [reply]
I am sry github ate the code part, fixed that in discription, it was late on a long day.... x.x

**Reply by TinMen6** [reply]
> I am sry github ate the code part, fixed that in discription, it was late on a long day.... x.x

don't wory about it, thanks for all the hard work
